src/main/resources/application.properties

```diff
@@ -1,23 +1,28 @@
 spring.application.name=eiffel-intelligence
-#${project.artifactId}
 
-# An extra optional version tag, version will be added after github release version 
+# An extra optional version tag, version will be added after GitHub release version.
 # Example if build.version=1.2.3-internal becomes Version 0.0.18 (1.2.3-internal)
 build.version=
 
-# port where Eiffel Intelligence will run
+# Port where Eiffel Intelligence will run
 server.port: 8090
 
-# path to file containing aggregation rules
-# if the path is a url it needs to include the http(s) protocol.
+# Path to file containing aggregation rules
+# If the path is a url it needs to include the http(s) protocol.
 rules.path: /rules/ArtifactRules-Eiffel-Agen-Version.json
+# The marker to be replaced with an Eiffel event ID extracted from IdentifyRules
+rules.replacement.marker: %IdentifyRulesEventId%
```

Comment:
Now when you rename this property, you maybe can rename all properties to follow Java/Spring properties format.
Or is too much work in this PR?

Reply:
I'd say i's too much work in this PR since I've already changed a lot just to adapt for this change 😄 There is a separate issue for fixing the rest of the properties